public/src/main/java/com/revenuecat/purchases/models/PurchaseOption.kt

```diff
@@ -1,36 +1,23 @@
 package com.revenuecat.purchases.models
 
 import android.os.Parcelable
-import com.revenuecat.purchases.parceler.JSONObjectParceler
-import kotlinx.parcelize.Parcelize
-import kotlinx.parcelize.TypeParceler
-import org.json.JSONObject
 
-@Parcelize
-@TypeParceler<JSONObject, JSONObjectParceler>()
-data class PurchaseOption(
+interface PurchaseOption : Parcelable {
     /**
      * Pricing phases defining a user's payment plan for the product over time.
      */
-    val pricingPhases: List<PricingPhase>,
+    val pricingPhases: List<PricingPhase>
 
     /**
-     * Tags defined on the base plan or offer in the Google dashboard.
+     * Tags defined on the base plan or offer. Empty for Amazon.
```

Comment:
Shouldn't this be only part of the `GooglePurchaseOption` then? I don't see the benefit of making this an interface if we have things that are not common to both stores in the interface. If GoogleStoreProduct just has `token`, should we then have it in the interface?

I know we talked about having here only stuff that is necessary for building a paywall without code that differs between stores, but this kinda breaks that right?

Reply:
Well, I think tags might be very useful/necessary for devs to choose an offer. Difficult to detect them otherwise, so I think the use case warrants it being more accesible in the interface,  without devs having to downcast it. So my vote would be to keep it here.

Reply:
ah true, since they won't have any offer IDs...

@vegaro i'm gonna go ahead and merge for now but open to reconsider if you want.

Reply:
I'd keep tags in the public API... it's so generic that we can feed that info with different sources for any other stores, so I think it won't hurt.

Reply:
Alright, you convinced me. Let's keep it in the interface